src/ert/_c_wrappers/job_queue/workflow.py

```diff
@@ -68,6 +69,8 @@ def from_file(
             content = parser.parse(
                 to_compile, unrecognized=UnrecognizedEnum.CONFIG_UNRECOGNIZED_ERROR
             )
+        except CombinedConfigError as err:
```

Comment:
Here we will have to replace the config_file location which has been set to a temporary directory, with `src_file`.

Reply:
Done, with this logic it is now also assumed that the config_file will be set for all the "child" errors of the `CombinedConfigError` raised from parsing, since the operation is setting the `config_file` for an already combined error (with potentially not the same file origin? I'm guessing not but don't know it for sure)

Reply:
A bit too magical indeed, this will be resolved by another future pass where the `config_file` parameter is deprecated and the logic is somehow unified for the two in a (ideally) consistent manner